homeassistant/components/flux_led/light.py

```diff
@@ -148,6 +155,7 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         device = {}
         device["name"] = device_config[CONF_NAME]
         device["ipaddr"] = ipaddr
+        device[CONF_UNIQUE_ID] = device_config.get(CONF_UNIQUE_ID)
```

Comment:
> Why not use the ip address as unique id?

Because that is not allowed.
<https://developers.home-assistant.io/docs/entity_registry_index/#unacceptable-sources-for-a-unique-id>